client/js/components/map/admin/AdminLayerListItem.vue

```diff
@@ -562,7 +562,7 @@ export default {
       if (this.layer.attributes.canUserToggleVisibility === false) {
         return Translator.trans('explanation.gislayer.visibility.group.locked.different.not.togglable')
       }
-      if (this.layer.attributes.visibilityGroupId !== this.activeLayerVisibilityGroupId || this.layer.attributes.visibilityGroupId !== '') {
+      if (this.layer.attributes.visibilityGroupId !== this.activeLayerVisibilityGroupId || this.layer.attributes.visibilityGroupId !== '' || this.layer.attributes.visibilityGroupId !== null) {
```

Comment:
I think storing these conditions in variables with descriptive names could increase the readability quite a bit, but I understand if there's no time for that..
Can't we just check here if `visibilityGroupId` is defined?

`if (this.layer.attributes.visibilityGroupId !== this.activeLayerVisibilityGroupId || !!this.layer.attributes.visibilityGroupId) {`